partials/tabs/projects.html

```diff
@@ -30,46 +30,44 @@
   <div id="modal1" class="modal">
     <div  class="modal-content-bear">
       <div class="col-md-4 profile">
-        <div class="project-name">
-          {{currentProject.name}}
+        <div btf-markdown="currentProject.name" class="project-name markdown">
         </div>
         <div ng-show="currentProject.requirements.length>0" class="project-detail-element  reqs-proj">
           <div class="small-heading">REQUIREMENTS</div>
-          <li class="pr-element-detail milestones" ng-repeat="req in currentProject.requirements">{{ req }}</li><br>
+          <li btf-markdown="req" class="pr-element-detail milestones markdown" ng-repeat="req in currentProject.requirements"></li><br>
         </div>
 
         <div ng-show="currentProject.initiatives.length>0" class="project-detail-element">
           <div class="small-heading">INITIATIVES</div>
-          <span class="pr-element-detail chip" ng-repeat="initiative in currentProject.initiatives">{{ initiative }}</span><br>
+          <span btf-markdown="initiative" class="pr-element-detail chip markdown" ng-repeat="initiative in currentProject.initiatives"></span><br>
         </div>
 
         <div ng-show="currentProject.tags.length>0" class="project-detail-element">
           <div class="small-heading">TAGS</div>
-          <span class="pr-element-detail chip" ng-repeat="tags in currentProject.tags">{{ tags }}</span>
+          <span btf-markdown="tags" class="pr-element-detail chip markdown" ng-repeat="tags in currentProject.tags">{{ tags }}</span>
         </div>
 
         <div ng-show="currentProject.difficulty" class="project-detail-element">
           <div class="small-heading">DIFFICULTY</div>
-          <span class="pr-element-detail chip">{{ currentProject.difficulty }}</span>
+          <span btf-markdown="currentProject.difficulty" class="pr-element-detail chip markdown"></span>
         </div>
 
         <div ng-show="currentProject.mentors.length>0" class="project-detail-element">
           <div class="small-heading">MENTORS</div>
-          <span class="pr-element-detail chip" ng-repeat="mentor in currentProject.mentors">@{{ mentor }}</span>
+          <span btf-markdown="mentor" class="pr-element-detail chip markdown" ng-repeat="mentor in currentProject.mentors">@</span>
         </div>
         <br>
       </div>
 
       <div class="col-md-8 dashboard">
-        <div class="project-description">
-          {{currentProject.desc}}
+        <div btf-markdown="currentProject.desc" class="project-description markdown">
         </div>
         <div class="row milestones-row">
           <div class="milestones medium-heading">MILESTONES</div>
           <div ng-repeat="milestone_header in currentMilestone">
-           <div class="small-heading">{{milestone_header}}</div>
+           <div btf-markdown="milestone_header" class="small-heading markdown"></div>
```

Comment:
this is also more of a term, the milestone descriptions may have markdown but not the names, they're not real text, just identifiers